dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -52,6 +52,7 @@ def __init__(self, options=None):
                              "of type NumericalOptions.")
         self.min = None
         self.max = None
+        self._top_k_modes = 5 # By default, return all modes
```

Comment:
are we assuming 5 is all modes by default?

Reply:
oops, need to update documentation